src/library_nodefs.js

```diff
@@ -1,5 +1,5 @@
 mergeInto(LibraryManager.library, {
-  $NODEFS__deps: ['$FS', '$PATH'],
+  $NODEFS__deps: ['$PATH'],
```

Comment:
> See conversation above, these were mentioned.

Thanks, and sorry I missed this, when rereading this was clear.



> So maybe the best thing is to do as you suggest in this PR, just remove deps from the fses to FS, and rely on FS being included when they are automatically anyhow. for that, we should probably also remove the FS dep from the other fses (even ones without postsets; for consistency), and we should write a comment in each of them instead of the deps, explaining this.

Yeah, ok to remove this to break the circular dependency, but please add a comment that mentions that logically these dependencies exist but they're circular, so that they are just omitted.